### ISSUE TITLE
The need for Versioned Dependencies or a Static Compilation method (musl)

### ISSUE THREAD

**Issue by 0pcom** [skycoin/skywire#614]
**Describe the bug**
In recent efforts to create .deb skywire packages on archlinux which reflect the recent changes in the develop branch, I ran into the following error when trying to run the compiled binaries on armbian

```
skywire-cli: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by skywire-cli)
```
I did not realise that glibc was a dependancy.

**Environment information:**

- OS: Armbian and ArchlinuxARM
- Platform:
Linux 5.8.16-sunxi64 aarch64 (armbian)
Linux 3.10.65+ aarch64 (archARM)

**Steps to Reproduce**

Compile the skycoin binaries with a newer version of glibc tan exists on the target system where the cmpiled binaries are installed. 

**Actual behavior**
The error :
```
skywire-cli: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by skywire-cli)
```

**Expected behavior**
skywire runs successfully

**Additional context**
This breaks my automated build, and so far alternatives have proven untenable.

Is there a list of the runtime dependancies that I might examine?

**Comment by 0pcom** [comment]
**possible implementation**

https://dominik.honnef.co/posts/2015/06/statically_compiled_go_programs__always__even_with_cgo__using_musl/

https://www.google.com/amp/s/ops.tips/amp/notes/glibc-golang-learnings/

**Comment by 0pcom** [comment]
This is fixed now if the PR gets approved.

**Comment by jdknives** [comment]
Fixed with #634
